plugins/GSdx/GSSetting.cpp

```diff
@@ -123,11 +123,12 @@ const char* dialog_message(int ID, bool* updateText) {
 			return "Makes textures partially or fully transparent as required by emulation. May cause unusual slowdowns for some games.";
 		case IDC_LOGZ:
 			return "Treat depth as logarithmic instead of linear. Recommended setting is on unless it causes graphical glitches.";
+		// Exclusive for Hardware Renderer	
+		case IDC_PRELOAD_GS:
+			return " Uploads GS data when rendering a new frame to reproduce some effects accurately. Fixes black screen issues in games like Primal.";
```

Comment:
Oops, I missed this one. Remove the space before Uploads.

Then I'll really merge it.